ui/app/src/services/content.ts

```diff
@@ -1482,27 +1483,33 @@ export function cut(site: string, item: SandboxItem): Observable<any> {
 
 export function paste(
   site: string,
-  item: SandboxItem
+  path: string
 ): Observable<{ site: string; status: string[] }> {
   return get(
-    `/studio/api/1/services/api/1/clipboard/paste-item.json?site=${site}&parentPath=${item.path}`
+    `/studio/api/1/services/api/1/clipboard/paste-item.json?site=${site}&parentPath=${path}`
   ).pipe(pluck('response'), catchError(errorSelectorApi1));
 }
 
 export function duplicate(
   site: string,
-  item: SandboxItem,
-  parentItem: SandboxItem
-): Observable<SandboxItem> {
+  path: string
+): Observable<string> {
+  let parentPath: any = path;
+  if (path.endsWith('index.xml')) {
+    parentPath = withoutIndex(path);
+  }
+  parentPath = parentPath.split('/');
+  parentPath.pop();
+  parentPath = parentPath.join('/');
```

Comment:
Create a getParentPath útil and use here

Reply:
done